Makefile

```diff
@@ -5,7 +5,7 @@ ifndef GOBIN
 endif
 
 build: prepare
-	$(GOBIN)/godep go build -o telegraf -ldflags \
+	$(GOBIN)/godep go build -race -o telegraf -ldflags \
```

Comment:
From the makefile, I think that I do. The packaging script does not use `make` to create it's binaries